ext/ftp/php_ftp.c

```diff
@@ -29,6 +29,7 @@
 
 #include "ext/standard/info.h"
 #include "ext/standard/file.h"
+#include "Zend/zend_attributes.h"
```

Comment:
I wonder if this include should be generated into the `arginfo.h`, because technically the function calls are within the arginfo.

Reply:
@kocsismate Not sure if you've missed this remark. Any comments with regard to this?

Reply:
No, I didn't miss it, just haven't answered it yet. I don't think we should add the include inside arginfo files. There are already a lot of other external code which is used inside these files, and none of them are included directly.

Reply:
Okay, that's fine for me.